glvis.cpp

```diff
@@ -160,7 +160,7 @@ bool GLVisInitVis(int field_type)
             vs->Zoom(1.8);
             // Use the 'bone' palette when visualizing a 2D mesh only (otherwise
             // the 'jet-like' palette is used in 2D, see vssolution.cpp).
-            paletteSet(4);
+            vs->GetPalette().SetPalette(4);
```

Comment:
This chain reads weird to me. Can it be just `vs->Palette()->Set(4)` or maybe an alternative shortcut `vs->SetPalette(4)`?

Reply:
I tried:
- removing `GetPalette()` for a `VisualizationScene::palette` global variable
- renaming `PaletteState::{Get,Set,Next,Prev}Palette` to `PaletteState::{Get,Set,Next,Prev}Index`

Let me know if that works for you.

Reply:
Looks good, thanks!